packages/core/src/stylable-processor.ts

```diff
@@ -492,7 +455,7 @@ export class StylableProcessor implements FeatureContext {
                     } else {
                         this.diagnostics.warn(
                             rule,
-                            processorWarnings.FORBIDDEN_DEF_IN_COMPLEX_SELECTOR(
+                            generalDiagnostics.FORBIDDEN_DEF_IN_COMPLEX_SELECTOR(
```

Comment:
`STImport.diagnostic.FORBIDDEN_DEF_IN_COMPLEX_SELECTOR`?

Reply:
After extracting this code to `STVars` (stylable vars), this code will change to refer to `STVars.diagnostics`

Reply:
I would left here a todo

Reply:
this is on my very close todo list with the rest of the symbol collecting features, it is written in my checklist